### ISSUE TITLE
Broken on lambda return x <> y

### ISSUE THREAD

**Issue by michaelborn** [jcberquist/commandbox-cfformat#25]
I have this line of code in my app - it's essentially stripping out query parameters which don't have an explicit value set.
```js
params = params.filter( ( key, param ) => param.value <> "<NOTSET>" );
```

CFFormat changed this to: (Notice the double spaces between `<` and `>`.

```js
params = params.filter( ( key, param ) => param.value <  > "<NOTSET>" );
```

I'm not sure what language construct CFFormat thinks that is, but it's definitely converting valid CFML to invalid CFML.

Is there somehow I can help fix this?

**Comment by jcberquist** [comment]
This is because the CFML syntax (from ST) I use to produce syntax scopes doesn't have a match for `<>` and so it sees this as two separate operators - I didn't even know that was valid syntax :). I will need to fix this "upstream". 

I see that Adam Cameron's cfscript reference has it: https://github.com/adamcameron/cfscript/blob/master/cfscript.md, but it is not referenced in cfdocs: https://cfdocs.org/operators

**Comment by michaelborn** [comment]
Interesting - I guess I've used that syntax so long I didn't realize it was Lucee only. I feel like it might be a good idea for me to change my code anyway to use `!=`. :)

**Comment by jcberquist** [comment]
This should be fixed in v0.9.0

**Comment by michaelborn** [comment]
You're amazing sir, thank you!
